src/components/MovieDetail/MovieDetail.js

```diff
@@ -2,22 +2,29 @@ import React from 'react';
 import './MovieDetail.css';
 
 const MovieDetail = ({movie}) => {
+
+  const formatDate = (movieReleaseDate) => {
+    const date = new Date(movieReleaseDate);
+    return date.toDateString();
+  };
+
+  const genres = movie.genres.join(' ');
+
```

Comment:
I like this a lot. I think maybe we should separate out more logic from the returns in these function components, and by we I mean me because I tend to keep everything in the return using jsx.